payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -0,0 +1,144 @@
+    fontSize = 14.sp
+)
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PaymentsColors(
+    val primary: Color,
+    val surface: Color,
+    val componentBackground: Color,
+    val componentBorder: Color,
+    val componentDivider: Color,
+    val onPrimary: Color,
+    val textSecondary: Color,
+    val placeholderText: Color,
+    val onBackground: Color,
+    val appBarIcon: Color,
+    val error: Color,
+)
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+object PaymentsThemeConfig {
+    fun colors(isDark: Boolean): PaymentsColors {
+        return if (isDark) colorsDark else colorsLight
+    }
+
+    private val colorsLight = PaymentsColors(
+        primary = Color(0xFF007AFF),
+        surface = Color.White,
+        componentBackground = Color.White,
+        componentBorder = Color(0x33787880),
+        componentDivider = Color(0x33787880),
+        onPrimary = Color.Black,
+        textSecondary = Color(0x99000000),
+        placeholderText = Color(0x993C3C43),
+        onBackground = Color.Black,
+        appBarIcon = Color(0x99000000),
+        error = Color.Red,
+    )
+
+    private val colorsDark = PaymentsColors(
+        primary = Color(0xFF0074D4),
+        surface = Color(0xff2e2e2e),
+        componentBackground = Color.DarkGray,
+        componentBorder = Color(0xFF787880),
+        componentDivider = Color(0xFF787880),
+        onPrimary = Color.White,
+        textSecondary = Color(0x99FFFFFF),
+        placeholderText = Color(0x61FFFFFF),
+        onBackground = Color.White,
+        appBarIcon = Color.White,
+        error = Color.Red,
+    )
+}
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PaymentsComposeColors(
+    val colorComponentBackground: Color,
+    val colorComponentBorder: Color,
+    val colorComponentDivider: Color,
+    val colorTextSecondary: Color,
+    val placeholderText: Color,
+    val material: Colors
+)
+
+@Composable
+@ReadOnlyComposable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun PaymentsThemeConfig.toComposeColors(): PaymentsComposeColors {
+    val colors = colors(isSystemInDarkTheme())
+    return PaymentsComposeColors(
+        colorComponentBackground = colors.componentBackground,
+        colorComponentBorder = colors.componentBorder,
+        colorComponentDivider = colors.componentDivider,
+        colorTextSecondary = colors.textSecondary,
+        placeholderText = colors.placeholderText,
+
+        material = lightColors(
+            primary = colors.primary,
+            onPrimary = colors.onPrimary,
+            surface = colors.surface,
+            onBackground = colors.onBackground,
+            error = colors.error,
+        )
+    )
+}
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun PaymentsTheme(
+    content: @Composable () -> Unit
+) {
+    val colors = PaymentsThemeConfig.toComposeColors()
+    val localColors = staticCompositionLocalOf { colors }
+
+    CompositionLocalProvider(
+        localColors provides colors
+    ) {
```

Comment:
How is localColors being used?  I think I had some existing code that had the style wrapped in classes like SectionTitle, and CardStyle, and I think it would be better to remove these classes and just use the variable from the compositionLocal. It seems more standard compose, it removes the extra layer of indirection, and makes it easy to see exactly where each color is being used in a find usages.

I would have expected something like:
```
    val LocalColors = staticCompositionLocalOf { colors }

    CompositionLocalProvider(
        LocalColors provides colors
    ) {
```

and then used like:

```@Composable
internal fun SectionTitle(@StringRes titleText: Int?) {
    titleText?.let {
        Text(
            text = stringResource(titleText),
            color = LocalColors.current.colorTextSecondary,
```